config/manager/manager.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: terraform-k8s
+  name: system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: operator
+  namespace: system
+  labels:
+    control-plane: terraform-k8s
+spec:
+  selector:
+    matchLabels:
+      control-plane: terraform-k8s
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: terraform-k8s
+    spec:
+      containers:
+      - command:
+        - /terraform-k8s
+        args:
+        - --enable-leader-election
```

Comment:
We'll need some additional args to watch the current namespace:

https://github.com/hashicorp/terraform-helm/blob/master/templates/sync-workspace-deployment.yaml#L65-L68